mrbelvedereci/plan/models.py

```diff
@@ -114,6 +129,18 @@ def check_push(self, push):
     ('hourly', 'Hourly'),
 )
 
+
+class PlanRepository(models.Model):
+    plan = models.ForeignKey(Plan)
+    repo = models.ForeignKey(Repository)
+
+    class Meta:
+        verbose_name_plural = 'PlanRepositories'
+
+    def __unicode__(self):
+        return u'{}:{}'.format(self.repo, self.plan)
```

Comment:
changed from `u'[{}] {}'` on purpose?

Reply:
No reason really, just that the other one was going away (plan.__unicode__) so I didn't see a need to keep it the same, but I don't really care so I'll change it back.